horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/datastore/CollectorApiDatastoreConfig.java

```diff
@@ -3,26 +3,30 @@
 import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
+import io.hyperfoil.tools.horreum.api.data.datastore.auth.APIKeyAuth;
+import io.hyperfoil.tools.horreum.api.data.datastore.auth.NoAuth;
+
 @Schema(type = SchemaType.OBJECT, required = true, description = "Type of backend datastore")
 public class CollectorApiDatastoreConfig extends BaseDatastoreConfig {
 
+    public static final String[] auths = { NoAuth._TYPE, APIKeyAuth._TYPE };
+    public static final String name = "Collectorapi";
+    public static final String label = "Collector API";
+    public static final Boolean builtIn = false;
```

Comment:
```suggestion
```

Reply:
This is accessed via reflection: https://github.com/Hyperfoil/Horreum/pull/2223/files#diff-8a4bd7c5a2730d0b066069e4fe1d81fd44eb3c7f84b69867ad74b83f74aca8deR27-R30